examples/conversion/conversionExamples.m

```diff
@@ -0,0 +1,71 @@
+%% Convert r1 struct to projectClass.
+% 
+% The following example shows how to convert r1 structs into projectClass.
+% 
+% r1 struct is converted into a projectClass using the r1ToProjectClass function.
+% 
+% The functions takes the name of the r1 struct .mat file.
+%
+% The function usage:
+
+projClass = r1ToProjectClass("originalDSPCBilayerStructInput.mat");
+
+
+%% Convert projectClass to r1 struct.
+% 
+% The following example shows how to convert projectClass into r1 struct.
+% 
+% projectClass is converted into r1 struct using the projectClassToR1 function.
+% 
+% The functions takes in a projectClass and other optional arguments.
+%
+% The optional arguments are:
```

Comment:
I think this information should be in the function docstring not the examples

Reply:
Moved.